<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AtomicConfigDialog</class>
    <widget class="QDialog" name="AtomicConfigDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>509</width>
                <height>574</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Atomic config</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="tab">
                        <attribute name="title">
                            <string>Fiat</string>
                        </attribute>
                        <layout class="QVBoxLayout" name="verticalLayout_4">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QListWidget" name="list_fiat"/>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="tab_2">
                        <attribute name="title">
                            <string>Crypto</string>
                        </attribute>
                        <layout class="QVBoxLayout" name="verticalLayout_5">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QListWidget" name="list_crypto"/>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QPushButton" name="btn_selectAll">
                            <property name="text">
                                <string>Select all</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_deselectAll">
                            <property name="text">
                                <string>Deselect all</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QDialogButtonBox" name="buttonBox">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="standardButtons">
                                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>AtomicConfigDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>AtomicConfigDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
